deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -27,7 +27,6 @@ metadata:
               "extraJvmOpts": "-Djava.property=me",
               "memory": "1Gi"
             },
-            "path": "asdasd"
```

Comment:
Isn't the field mandatory and we should change the `asdasd` to some more meaningful example default? @ryanemerson?

Reply:
@Crumby `path` is not part of the `Backup.Spec`.